specification/cognitiveservices/data-plane/EntitySearch/readme.nodejs.md

```diff
@@ -6,7 +6,7 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 ``` yaml $(nodejs)
 nodejs:
   package-name: azure-cognitiveservices-entitysearch
-  package-version: 1.0.0-preview
+  package-version: 1.1.0
```

Comment:
Note that the version we've actually published is 1.1.0--not likely to publish a new one as it looks like swagger has not changed. Just doing a correction here